lookout/style/format/analyzer.py

```diff
@@ -107,7 +108,9 @@ def group_line_nodes(y, y_pred, vnodes_y, rule_winners):
                     log.warning("Failed to parse %s", file.path)
                     continue
                 X, y, vnodes_y, vnodes = res
-                y_pred, rule_winners = rules.predict(X, vnodes_y, vnodes, fe)
+                client = BblfshClient(self.config["bblfsh_address"])
```

Comment:
We should create the client only once per `analyze()` call, in the outermost scope.